source/numerics.tex

```diff
@@ -16034,3 +16034,2479 @@
                                      A, t, d, B, divides<void>{});
 \end{codeblock}
 \end{itemdescr}
+
+\rSec1[simd]{Data-parallel types}
+\rSec2[simd.general]{General}
+
+\newcommand\foralli[1][]{for all $i$ in the range of \range{0}{#1size()}}
+\newcommand\Foralli[1][]{For all $i$ in the range of \range{0}{#1size()}}
+\newcommand\forallmaskedi{for all selected indices $i$ of \tcode{mask}}
+
+\newcommand\ConstraintUnaryOperatorWellFormed[2][const ]{%
+  \constraints \tcode{requires (#1value_type a) \{ #2; \}} is \tcode{true}.
+}
+
+\newcommand\ConstraintOperatorTWellFormed{%
+  \constraints \tcode{requires (value_type a, value_type b) \{ a \placeholder{op} b; \}} is \tcode{true}.
+}
+
+\pnum
+[simd] defines data-parallel types and operations on these types.
+\begin{note}
+The intent is to support acceleration through data-parallel execution resources
+where available, such as SIMD registers and instructions or execution units
+driven by a common instruction decoder.
+\end{note}
+
+\pnum
+The set of \defn{vectorizable types} comprises all standard integer types,
+character types, and the types \tcode{float} and \tcode{double}
+([basic.fundamental]).
+In addition, \tcode{std::float16_t}, \tcode{std::float32_t}, and
+\tcode{std::float64_t} are vectorizable types if defined ([basic.extended.fp]).
+
+\pnum
+The term \defn{data-parallel type} refers to all enabled specializations of
+the \tcode{basic_simd} and \tcode{basic_simd_mask} class templates. A \defn{data-parallel object} is
+an object of \term{data-parallel type}.
```

Comment:
Defined phrases are italicized once when defined, and not when referred to. Please remove \term here.